sqlglot/generator.py

```diff
@@ -320,7 +332,7 @@ class Generator:
     STRICT_STRING_CONCAT = False
     NORMALIZE_FUNCTIONS: bool | str = "upper"
     NULL_ORDERING = "nulls_are_small"
-    ESCAPE_LINE_BREAK = False
+    UNESCAPE_SEQUENCES = False
```

Comment:
could probably remove this in favor of whether or not this mapping exists?

Reply:
sure

Reply:
made these changes